### PR TITLE
README update

### PR DESCRIPTION
**Pull request by edevenport**
The README has been updated to include separate sections for block and object stores.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -55,6 +64,98 @@ The following will describe how to install and configure the DigitalOcean blocks
...
+    ```
+    kubectl -n heptio-ark edit deployment ark
+    ```
+
+    A full Deployment YAML example defining the Secret can be found in `examples/20-deployment.yaml`.
+
+5. Finally, add the `ark-blockstore-digitalocean` plugin to Ark.
+
+    ```
+    ark plugin add quay.io/stackpoint/ark-blockstore-digitalocean:latest
```
Comment (rosskukulinski) [inline README.md]: Should this be `:v0.1.0`?

**Comment by rosskukulinski** [comment]
This looks great to me.  @skriss or @nrb: do you have any additional comments?

File: README.md [hunk 2]
```diff
@@ -55,6 +64,98 @@ The following will describe how to install and configure the DigitalOcean blocks
...
+        --from-file cloud=./credentials-ark
+    ```
+
+5. Now add the Ark `backupStorageProvider` to the Ark default Config.
+
+    ```
+    kubectl -n heptio-ark edit config default
+    ```
+
+    Below is a sample `persistentVolumeProvider` YAML Config section. Be sure to change the bucket and region placeholder values accordingly.
```
Comment (skriss) [inline README.md]: s/persistentVolumeProvider/backupStorageProvider

**Comment by skriss** [comment]
Otherwise looks great!

Also just a heads-up (this has probably been communicated already) that a lot of this is going to be changing as of v0.10.  We can get you a pre-release version when we have one so you can check out the updates and revise docs & code.

File: README.md (hunk 1 above)
Comment (edevenport) [reply README.md]: I already moved the `latest` tag to `v0.1.0`. I think in most situations, we want to advise users to pull `latest` as that should be the latest version.

File: README.md (hunk 2 above)
Comment (edevenport) [reply README.md]: Good catch. That's been changed now.

**Comment by edevenport** [comment]
Excellent, I'm excited to checkout Ark v0.10 and will look at updating the plugin once I have access to the pre-release.

**Pull request merged**